browser/brave_wallet/brave_wallet_context_utils.h

```diff
@@ -6,11 +6,13 @@
 #ifndef BRAVE_BROWSER_BRAVE_WALLET_BRAVE_WALLET_CONTEXT_UTILS_H_
 #define BRAVE_BROWSER_BRAVE_WALLET_BRAVE_WALLET_CONTEXT_UTILS_H_
 
-class Profile;
+namespace content {
+class BrowserContext;
+}
 
 namespace brave_wallet {
 
-bool IsAllowedForProfile(Profile* profile);
+bool IsAllowedForContext(content::BrowserContext* context);
```

Comment:
do you think we should move this to `components/brave_wallet`?

Reply:
yeah we can definitely do so.

Reply:
https://github.com/brave/brave-browser/issues/15177